src/modules/tree.ts

```diff
@@ -137,6 +137,18 @@ export default {
         type: 'boolean',
       },
     ],
+    typings: {
+      description: 'Download react and graphql app typings',
+      handler: './setup/typings',
+    },
+    tooling: {
+      description: 'Setup linter and formatter',
+      handler: './setup/tooling',
+    },
+    tsconfig: {
+      description: 'Setup tsconfig',
+      handler: './setup/tsconfig',
+    },
```

Comment:
🤔 
I think I would always want to run `vtex setup tooling` and `vtex setup tsconfig`.

So maybe what do you think of flags so we are able to compose them?

`vtex setup --tooling --tsconfig`

Reply:
Do you think only the flags would be sufficient?
@kaisermann @lucasecdb @klzns

Reply:
I think so, maybe we could add a "preset" for both the tooling and tsconfig flags

Reply:
What do you guys think:
```
--all
--tsconfig
--tooling
--typings
```
@lucasecdb that would be nice, but I don't know which flag would fit

Reply:
I don't have a good name idea for that preset.

Anyway, I think the help should be clear which tools it installs with the --tooling command.

Also, we need to figure it out why some projects typings dont work so we wouldn't be needing this feature.

Reply:
I think if it starts to be too much complicated on the choices the subcommands are the way to go, so `vtex setup tooling` would have flags for `--prettier`, `--eslint`, `--husky`

@klzns Actually this feature is because some devs don't want the tooling installed in their projects - most web-framework doesn't wan't it

Reply:
I think only the flag --tooling is fine. I'm just considering writing that it will:

> Installs Eslint, Prettier and pre commit hooks.

In the description of it when you run  --help.

Reply:
Perfect! Sorry, I misread up there